lib/src/database/matrix_sdk_database.dart

```diff
@@ -36,6 +36,17 @@ import 'package:matrix/src/utils/run_benchmarked.dart';
 import 'package:matrix/src/database/indexeddb_box.dart'
     if (dart.library.io) 'package:matrix/src/database/sqflite_box.dart';
 
+/// Database based on SQlite3 on native and IndexedDB on web. For native you
+/// have to pass a `Database` object, which can be created with the sqflite
+/// package like this:
+/// ```dart
+/// final database = await openDatabase('path/to/your/database');
+/// ```
+///
+/// **WARNING**: For android it seems like that the CursorWindow is too small for
+/// large amounts of data if you are using SQFlite. Consider using a different
+/// library. Learn more at:
```

Comment:
```suggestion
/// large amounts of data if you are using SQFlite. Consider using a different
/// package to open the database like [sqflite_sqlcipher](https://pub.dev/packages/sqflite_sqlcipher) or https://pub.dev/packages/sqflite_common_ffi. Learn more at:
```

I think it is slightly confusing, if they should now not use the matrix sdk or not use sqlite or not use sqflite. Not sure how to improve that wording though and maybe it was just confusing because I read the "how to use this library" readme just before this in the context of this PR.

Reply:
changed :) thanks